.gitignore

```diff
@@ -6,3 +6,6 @@ npm-debug.log
 .tern-project
 yarn-error.log
 .vscode/
+manifest.yml
+.imdone/
+config.local.yml
```

Comment:
I personally think it's fine. `manifest.yml` is a Bluemix artifact, and `.imdone` is from Atom. A lengthy `.gitignore` isn't a bad thing if it improves the developer experience for contributors.

@tortilaman what is `config.local.yml` from?